src/components/Dropdown/DropdownError.st.css

```diff
@@ -0,0 +1,38 @@
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: Tooltip;
+}
+
+:import {
+    -st-from: "../Tooltip/Tooltip.st.css";
+    -st-default: TPATooltip;
+}
+
+.root {
+    -st-extends: Tooltip;
+    -st-mixin: TPATooltip;
+    color: value(errorColor);
+    box-sizing: border-box;
+    position: absolute;
+    top: 50%;
+    transform: translate(0,-50%);
+    width: 100%;
+    padding-inline-end: 32px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    cursor: pointer;
```

Comment:
Are all these needed?
We don't get this from the tooltip component?

Reply:
Without them it looks like this:
![image](https://user-images.githubusercontent.com/47482250/67382600-6f236180-f596-11e9-823c-199636c4150b.png)